prow/prowjobs/kubeflow/pipelines/kubeflow-pipelines-presubmits.yaml

```diff
@@ -171,6 +171,16 @@ presubmits:
         command:
         - ./test/presubmit-components-gcp.sh
 
+  - name: kubeflow-pipelines-components-google-cloud-python38
+    cluster: build-kubeflow
+    decorate: true
+    run_if_changed: "^(components/google-cloud/.*)|(/pipelines/components/google-cloud/tests/presubmit.sh)$"
```

Comment:
Presubmit test script is already in the first folder, so can we simplify?

Reply:
good point, done :)